test/controllers/invoices_controller_test.rb

```diff
@@ -0,0 +1,38 @@
+require 'test_helper'
```

Comment:
Please remove all these test files and try not to use generators at this point because it generates unnecessary files.
We wont be writing test cases at this point.
@Salahuldin Please take note of this as well.